src/core/a-entity.js

```diff
@@ -266,12 +264,17 @@ var proto = Object.create(ANode.prototype, {
           // For scene default components, expose them in the DOM.
           HTMLElement.prototype.setAttribute.call(this, name, this.defaultComponents[name]);
         }
+
+        // Check if component already defined.
+        if (name in this.components) { return; }
```

Comment:
Why is this necessary? isComponentDefined should catch if the component is already defined.